pkgs/applications/science/math/pagsuite/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, stdenv
+, fetchzip
+, cmake
+, gmp
+, scalp
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pagsuite";
+  version = "1.80";
+
+  src = fetchzip {
+    url = "https://gitlab.com/kumm/${pname}/-/raw/master/releases/${pname}_${builtins.replaceStrings ["."] ["_"] version}.zip";
+    sha256 = "sha256-JuRuDPhKKBGz8jUBkZcZW5s2berOewjsPNR/n7kuofY=";
+    stripRoot = false;
+    postFetch = ''
+      mv $out/${pname}*/* $out
+    '';
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  buildInputs = [
+    gmp
+    scalp
+  ];
+
+  enableParallelBuilding = false;
```

Comment:
Can you add a short comment why we want to disable this.

Reply:
I can no longer reproduce the build failure, so I removed it.